src/main/java/com/github/manolo8/darkbot/backpage/NativeBrowserImpl.java

```diff
@@ -0,0 +1,41 @@
+package com.github.manolo8.darkbot.backpage;
+
+import com.github.manolo8.darkbot.extensions.util.Version;
+import com.github.manolo8.darkbot.gui.titlebar.BackpageTask;
+import eu.darkbot.api.managers.NativeBrowserAPI;
+import org.jetbrains.annotations.Nullable;
+
+import java.io.IOException;
+
+public class NativeBrowserImpl implements NativeBrowserAPI {
+
+    private final BackpageManager backpageManager;
+
+    public NativeBrowserImpl(BackpageManager backpageManager) {
+        this.backpageManager = backpageManager;
+    }
+
+    @Override
+    public boolean isSupported() {
+        return BackpageTask.isSupported(new Version("1.3.0"));
+    }
+
+    @Override
+    public @Nullable Process openLink(String link) {
+        if (isSupported() && backpageManager.isInstanceValid()) {
+            try {
+                return BackpageTask.createBrowser("--sid " + backpageManager.getSid(),
+                        "--url " + backpageManager.getInstanceURI().toString(), // for sid
+                        "--fullurl " + link);
```

Comment:
This makes no sense, if you expect the SID to be passed i'd use a openGameLink, if you are opening any other link it'd expect it not to have sid passed, am i wrong?